docs/monitoring-usage.rst

```diff
@@ -278,3 +278,91 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instance of
+:class:`~gcloud.monitoring.metric.Metric` created with a metric type as well as values for
+the labels associated with that metric type. The metric type can either
```

Comment:
Is it inaccurate to say you create the type by creating a metric descriptor? Auto-creation docs read similar to that: 

https://cloud.google.com/monitoring/custom-metrics/creating-metrics#auto-creation

"If you write metric data to a custom metric type that does not yet exist, then a new custom metric type is created for you automatically"